core/chain-configs/src/updateable_config.rs

```diff
@@ -98,5 +98,6 @@ pub struct UpdateableClientConfig {
     pub resharding_config: ReshardingConfig,
 
     /// Time limit for adding transactions in produce_chunk()
+    #[serde(with = "near_async::time::serde_opt_duration_as_std")]
```

Comment:
quick question, do we need this for client config? I see that the standard client config class has a lot of duration related members.

Reply:
Maybe your fix for including serde feature with time module fixed this?

Reply:
ClientConfig serde is needed for printing it out in the debug page I think. It's not very critical what exact form it is in.